src/main/java/lotto/constant/Rank.java

```diff
@@ -0,0 +1,42 @@
+package lotto.constant;
+
+import java.util.Optional;
+
+public enum Rank {
+    FIRST(6, 0, 2000000000),
```

Comment:
숫자의 자리수가 많아질 때는 `_`로 자리수를 구분해 줄 수 있습니다!
```suggestion
    FIRST(6, 0, 2_000_000_000),
```